config/puma.rb

```diff
@@ -42,3 +42,12 @@
 
 # Allow puma to be restarted by `rails restart` command.
 plugin :tmp_restart
+
+pidfile "/tmp/puma.pid"
+state_path "/tmp/puma.state"
+
+if ENV["VELUM_PORT"].to_i == 443
+  ssl_bind "0.0.0.0", ENV["VELUM_PORT"], key: "/etc/pki/velum.key", cert: "/etc/pki/velum.crt"
```

Comment:
I think in this case we can just stick to `0.0.0.0`, I agree with @kiall that this can be improved, but we can do this at a later time. It's not ideal, but when we change this to a specific interface we should also consider generate the velum certificate with the chosen IP address only.